airbyte-integrations/bases/base-normalization/integration_tests/dbt_integration_test.py

```diff
@@ -82,7 +82,7 @@ def setup_postgres_db(self):
         ]
         print("Executing: ", " ".join(commands))
         subprocess.call(commands)
-        time.sleep(120)
+        time.sleep(5)
```

Comment:
Opened an issue here to change this in the future: https://github.com/airbytehq/airbyte/issues/6048